Packs/Exabeam/Integrations/Exabeam/Exabeam.py

```diff
@@ -694,10 +696,48 @@ def get_notable_sequence_event_types_request(self, asset_sequence_id: str = None
                                       params=params)
         return response
 
+    def get_list_incidents(self, query_params: dict):
+
+        return self._http_request('GET', url_suffix='/ir/api/incident/list',
+                                  params=query_params)
+
+    def get_single_incident(self, incident_id: str, username: str = None):
+        headers = self._headers | {'EXA_USERNAME': username or self.username}
+        return self._http_request('GET', url_suffix=f'/ir/api/incident/{incident_id}',
+                                  headers=headers)
+
+    def get_incidents(self, query: dict[str, Any]):
+
+        return self._http_request(
+            'POST',
+            url_suffix='/ir/api/incidents/search',
+            json_data=query,
+        )
+
 
 ''' HELPER FUNCTIONS '''
 
 
+def format_single_incident(incident: dict[str, Any]) -> dict[str, Any]:
+    incident_fields = incident.get('fields', {})
+    formatted_incident = {'incidentId': incident.get('incidentId'),
+                          'name': incident.get('name'),
+                          'fields': {
+        'startedDate': convert_unix_to_date(incident_fields.get('startedDate')),
+        'closedDate': convert_unix_to_date(incident_fields.get('closedDate')),
+        'createdAt': convert_unix_to_date(incident_fields.get('createdAt')),
```

Comment:
are these values (`startedDate` etc) always present? 

If so, please use `['startedDate']`.
Otherwise, how do we handle `None` values?

Reply:
This is handled by the function `convert_unix_to_date`